src/tests/api/alarms/BavAlarms.test.ts

```diff
@@ -0,0 +1,91 @@
+/* eslint-disable max-lines-per-function */
+import { abortPost, authorizationGet, personInfoGet, sessionPost, startStubServiceAndReturnSessionId, stubStartPost, tokenPost, userInfoPost, verifyAccountPost } from "../ApiTestSteps";
+import { sleep } from "../../../../src/utils/Sleep";
+import { describeAlarm } from "../ApiUtils";
+import verifyAccountYesPayload from "../../data/bankDetailsYes.json";
+import { BankDetailsPayload } from "../../models/BankDetailsPayload";
+import { randomUUID } from "crypto";
+
+describe("BAV CRI Alarms Tests", () => {
+
+	it("/session Endpoint 4xx Api Gateway Alarm", async () => {
+		for (let i = 1; i <= 250; i++) {
+			const stubResponse = await stubStartPost();
+			await sessionPost("", stubResponse.data.request);
+		}
+		await sleep(300000);
+
+		const alarm = await describeAlarm("bav-cri-api-Session4XXApiGwErrorAlarm");
+		console.log("Session Endpoint Alarm State: " + alarm.StateValue);
```

Comment:
Are these console logs for the devs benefit?

Reply:
It was initially, but I should've updated to assertions. Pushing that change now